src/views/Swap/components/CurrencyInputHeader.tsx

```diff
@@ -47,10 +61,10 @@ const CurrencyInputHeader: React.FC<Props> = ({ title, subtitle, setIsChartDispl
         </Flex>
         <Flex>
           <NotificationDot show={expertMode}>
-            <GlobalSettings color="primary" mr="0" />
+            <GlobalSettings color="textSubtle" mr="0" />
           </NotificationDot>
           <IconButton onClick={onPresentTransactionsModal} variant="text" scale="sm">
-            <HistoryIcon color="primary" width="24px" />
+            <HistoryIcon color="textSubtle" width="24px" />
```

Comment:
Why color change? Seems like all icons have primary color in Figma 👀

Reply:
Yuna asked for it 🙂